data/operators/advertising/billboard.json

```diff
@@ -570,8 +570,58 @@
     },
     {
       "displayName": "IKEA",
-      "id": "ikea-d4816c",
-      "locationSet": {"include": ["001"]},
+      "id": "ikea-1dde13",
+      "locationSet": {
+        "include": [
```

Comment:
I guess this is unnecessary? because the wholesale market IKEA is truly have business in too many countries, so the advertising board can be seen in those so many countries, I guess it is also OK to keep the 001 and don't need to list such long list?

Reply:
Oh my fault, just noticed that currently all IKEA related things are described in a full list. 🥺